modules/core/04-channel/types/query.go

```diff
@@ -94,6 +94,18 @@ func NewQueryNextSequenceReceiveResponse(
 	}
 }
 
+// NewQueryUpgradeErrorResponse creates a new QueryUpgradeErrorResponse instance
+func NewQueryUpgradeErrorResponse(sequence uint64, errReceipt string, proof []byte, height clienttypes.Height) *QueryUpgradeErrorResponse {
```

Comment:
I think we should accept the `ErrorReceipt` as an arg rather than the sequence and error string